docker/test/performance-comparison/config/config.d/top_level_domains_path.xml

```diff
@@ -0,0 +1,3 @@
+<yandex>
+    <top_level_domains_path replace="replace">/top_level_domains/</top_level_domains_path>
```

Comment:
Why another file and not just add it to tweaks? This looks like an absolute path which does not exist, is it going to work? The configs end up at `{right,left}/db/config/top_level_domains`, not sure how we can specify this path...

Reply:
https://github.com/ClickHouse/ClickHouse/pull/11711

There is something weird going on with these paths... maybe we'll have to add another command line argument.

Reply:
>Why another file and not just add it to tweaks?

For no reason

> This looks like an absolute path which does not exist, is it going to work? The configs end up at {right,left}/db/config/top_level_domains, not sure how we can specify this path...

It does, since it copies everything inside the container from the `docker/test/performance-comparison/config/`

https://github.com/ClickHouse/ClickHouse/blob/f6f336ac7839ed52a147eb0bcae8b88f0548082a/docker/test/performance-comparison/Dockerfile#L40

But it does not dereference symlinks, and docker will not do this, since it cannot leave the context, will address this

>There is something weird going on with these paths... maybe we'll have to add another command line argument.

Interesting. But this overrides will work, since it is absolute path, but ok, let's move it to the command line arguments.